pkg/bloombuild/planner/testutils/utils.go

```diff
@@ -0,0 +1,137 @@
+package testutils
```

Comment:
extremely tiny nit: Generally I prefer package names being slightly more [descriptive](https://go.dev/blog/package-names#bad-package-names) about their domain than utils/testutils. 

`plannerfakes`, `plannerutils`, or `testplanner` would all be slightly more  descriptive here, in the same vein as the stdlib [slogtest package](https://pkg.go.dev/testing/slogtest@go1.23.2).

That being said, of my opinions about Go, I feel the least strongly about this one, so I don't mind if this stays the same, either.